ui/src/components/table/CustomTableRow.jsx

```diff
@@ -73,6 +74,21 @@ function CustomTableRow(props) {
                             />
                         </Tooltip>
                     )}
+                    {rowActions.includes('search') && (
+                        <Tooltip content={_('Search')}>
```

Comment:
maybe `Go to search for events associated with ${selectedRow.name}`  would be more clear

Reply:
Done